resources/defaults.css

```diff
@@ -21,7 +21,7 @@ body {
   will-change: content;
 }
 
-.lang-ja .text {
+:lang(ja) .text {
```

Comment:
Should this be `ja-JP`?

Reply:
As[MDN](https://developer.mozilla.org/ja/docs/Web/HTML/Global_attributes/lang) says, `:lang(ja)` matches all valid language code that prefixed by `ja`, so it is okay here.

As for Chinese, Simplified Chinese and Traditional Chinese use different characters set and different words, so it is nessessary to differentiate `zh-CN` and `zh-TW` or something else.